### ISSUE TITLE
False positive no-member with sys.stdin.buffer.peek()

### ISSUE THREAD

**Issue by esabouraud** [PyCQA/pylint#3728]
### Steps to reproduce
1. Save code as `stdin_peek.py`
2. Run` python3 -m pylint stdin_peek.py`
```
"""Test case for E1101 false positive"""

import sys

def stdin_peek():
    """pylint objects to the usage of peek() here"""
    return sys.stdin.buffer.peek(16)
```

### Current behavior
Current pylint output:
```
************* Module stdin_peek
stdin_peek.py:7:11: E1101: Instance of 'BufferedWriter' has no 'peek' member; maybe 'seek'? (no-member)
```

### Expected behavior
No `no-member` violation.

`sys.stdin.buffer` type is BufferedReader, which does have a 'peek' member.
```
>>> import sys
>>> type(sys.stdin.buffer).__name__
'BufferedReader'
```

### pylint --version output
```
pylint 2.5.3
astroid 2.4.2
Python 3.8.0 (tags/v3.8.0:fa919fd, Oct 14 2019, 19:37:50) [MSC v.1916 64 bit (AMD64)]
```
